### ISSUE TITLE
Adding a New Example For A Minimal gRPC + WebApi Project

### ISSUE THREAD

**Issue by RyanCallahan312** [grpc/grpc-dotnet#1517]
Would it be possible to add a new example to the examples directory that combines the a the Weather Forecast WebApi template and gRPC server? I know the [Microservicer](https://github.com/grpc/grpc-dotnet/tree/master/examples/Microservicer) example is similar to this, but it does it with razor pages in a separate project. I believe there are some use cases where it would be preferable to have REST controllers with gRPC endpoints in the same project.

I recently created [this repo](https://github.com/RyanCallahan312/dotnet-6-gRPC-and-REST-api) that is exactly that with docker support. I found the the existing information about getting this setup outdated in some places and believe that if this example was in the repo it could have benefited me.

If this would be something that gets approval, I would be interested in putting out a PR for it if that's ok.

**Comment by JamesNK** [comment]
Hi

I'm not sure how valuable having that as an example is. Registering WebAPI services and mapping controllers isn't complicated. And it's not demoing a feature in gRPC.

**Comment by RyanCallahan312** [comment]
True its not really demoing a feature, but I found that having both the grpc service and web controllers registered and not interfering with each other was the trickiest part and then also the kestrel config was another snag. I was mostly running into the issue of everything being outdated articles/stackoverflow questions and this use case not being shown at all in the examples. The closest thing to it I saw is microservicer having mvc in a separate project from the grpc server.

Looking back on it I remember that's what gave me all the issues I had, but I can't remember exactly my step by step process on getting it working as I've been cramming for college finals the last week. 

If this isn't something that would fit into the examples that's fine, I thought that would be the most effective place to put it for anyone who might be looking to do something similar. I'm just looking to prevent others from running into the same issues that I did.
